apps/core/admin.py

```diff
@@ -1 +1,15 @@
-# Register your models here.
+from django.contrib import admin
+from .models import Species, Strain, OmicsUnitType, \
+                    OmicsUnit, Pixel, Tag, Experiment, \
+                    Analysis, OmicsArea, Pixeler
```

Comment:
General remark about imports: i. I prefer using the tuple syntax, ii. with imports sorted alphabetically:

```python
from .models import (
    Analysis, Experiment, OmicsArea, OmicsUnit, OmicsUnitType, Pixel, Pixeler, 
    Species, Strain, Tag
)
```

But, in this case, if you will use all models, why not importing the `models` module itself?

```python
from . import models

admin.site.register(models.Analysis)
```

Reply:
The enumeration is part of the good practices in my remembers.

Reply:
Nope. Particularly when you will use a lot of module's content.